logging/logging.go

```diff
@@ -5,13 +5,19 @@ import (
 	"fmt"
 	"os"
 	"strings"
+	"sync"
+	"time"
 
 	"golang.org/x/net/context"
 
 	log "github.com/Sirupsen/logrus"
 	"github.com/weaveworks/common/user"
 )
 
+const (
+	DEFAULT_DEDUPE_INTERVAL = time.Minute
```

Comment:
./logging/logging.go:18:2: don't use ALL_CAPS in Go names; use CamelCase
./logging/logging.go:18:2: exported const DEFAULT_DEDUPE_INTERVAL should have comment (or a comment on this block) or be unexported